src/silx/gui/plot/LegendItem.py

```diff
@@ -0,0 +1,156 @@
+import logging
+
+from .. import qt
+from ..widgets.LegendIconWidget import LegendIconWidget
+from . import items
+from .items.core import HighlightedMixIn
+from .PlotWidget import PlotWidget
+
+_logger = logging.getLogger(__name__)
+
+
+class LegendItemWidget(qt.QWidget):
+    def __init__(self, parent, item: items):
```

Comment:
`items` is a module, not  a class
```suggestion
    def __init__(self, parent, item: items.Item):
```